src/components/Profile/Albums/Albums.jsx

```diff
@@ -96,8 +95,14 @@ class Albums extends React.Component {
   }
 
   loadAlbums = async () => {
-    const allAlbums = await queries.getAlbums();
-    this.setState({ albums: allAlbums });
+    const { isMainPage } = this.props;
+    if (isMainPage) {
+      const allAlbums = await queries.getAllAlbums();
+      this.setState({ albums: allAlbums });
+    } else {
+      const userAlbums = await queries.getAlbums();
+      this.setState({ albums: userAlbums });
+    }
```

Comment:
const albums = isMainPage ? await queries.getAllAlbums() : await queries.getAlbums()
this.setState({ albums })